frontend/public/models/index.ts

```diff
@@ -598,22 +585,6 @@ export const RoleModel: K8sKind = {
   labelPluralKey: 'public~Roles',
 };
 
-export const SelfSubjectAccessReviewModel: K8sKind = {
```

Comment:
can let this copy be here as is and not export from SDK, as in SDK we should avoid having models. If needed for required one's we can have bare minimal if needed and this has translation stuff